looprpc/server.proto

```diff
@@ -74,4 +78,4 @@ message ServerLoopInQuoteResponse {
     uint64 min_swap_amount = 4;
     uint64 max_swap_amount = 5;
```

Comment:
I think we should already now add a `LoopOutTerms` call to the server to expose the static terms (at least `min_swap_amount` and `max_swap_amount`). Otherwise someone willing to find out the min/max range needs to guess an amount within that range first to get a valid quote response.

Reply:
Added